manifests/e/EclipseAdoptium/Temurin/11/JRE/11.0.20.8/EclipseAdoptium.Temurin.11.JRE.installer.yaml

```diff
@@ -3,14 +3,17 @@
 
 PackageIdentifier: EclipseAdoptium.Temurin.11.JRE
 PackageVersion: 11.0.20.8
+ReleaseDate: 2023-07-21
 InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0

```

Comment:
```suggestion
```

The `MinimumOSVersion` is not needed.

Reply:
That's a fair point that I did consider very briefly. I am wondering whether removing this (or keeping it) actually does anything? Is there any client side software using the winget repository NOT on >= Win 10? I can absolutely update the definitions, no problem.

Why did I add it in the first place? This is an amalgamation of all the other definitions, some of which already had it, and ultimately I think I should have validated some of these values a little more I guess.

Reply:
So, the 8.1 version string would be `6.3.9600` or is omitting it the better idea altogether?

Reply:
> Is there any client side software using the winget repository NOT on >= Win 10?

I don't think so but even then the entry is redundant.

> This is an amalgamation of all the other definitions, some of which already had it

Manifests created with yamlcreate add that entry but I don't know why because it shouldn't be needed. If it is needed, the pipelines will give a https://github.com/microsoft/winget-pkgs/labels/Manifest-Installer-Validation-Error - I've only had this occur for MSIXs that have that hard minimum requirement on a specific Windows 10 version.

> So, the 8.1 version string would be `6.3.9600` or is omitting it the better idea altogether?

Omitting it would be best as there shouldn't be anything running winget below Windows 10 and that minimum Temurin has on their website is probably for support - it could well work on Windows 7, for example.